README.md

```diff
@@ -262,11 +262,22 @@ assistant.message(params).then(
     console.log(response.headers);
   },
   err => {
-    console.log('error: ', err);
+    console.log(err);
+    /*
+      `err` is an Error object. It will always have a `message` field
+      and depending on the type of error, it may also have the following fields:
+      - body
+      - headers
+      - name
+      - code
+    */
   }
 );
 ```
 
+### Global Transaction ID
+Every SDK call returns a response with a transaction ID in the x-global-transaction-id header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.
+
```

Comment:
Can you add a code sample here to explicitly show how to get the header?

Reply:
For sure

Reply:
Code samples added for HTTP and WebSocket methods